### DISCUSSION TITLE
Error when trying to install version 1.0.3

### DISCUSSION THREAD

**Discussion by EricdeWaal** [akeeba/panopticon#268] [Q&A (Community Support)] [answered]
Help keep Issues free from distractions so that Panopticon developers can work more efficiently. If you cannot provide accurate reproduction instructions please open a new Q&A Discussion instead: https://github.com/akeeba/panopticon/discussions/new?category=q-a-community-support

## Describe the bug

After the first Welcome to Panopticon screen, a click on Let's Go results in a red screen, with some errors:
Value error link > Missing format specifier at end of string, 
HTTP 500 , 
Followed a part of code visible

## Reproduction instructions

Fresh install on server meeting all requirements, in a subdomain. Unzip panopticon-1.0.3.zip in the subdomain's directory

## Expected behaviour
Next configuration step

## Screenshots
![afbeelding](https://github.com/akeeba/panopticon/assets/5586423/64883791-53d9-49e3-b28a-07d95579e8fa)

## Environment (please complete the following information)
 - Panopticon version: 1.03
 - PHP version: 8.1
 - Database type MariaDB
 - Database version: 10.5
 - Browser: both Firefox and Edge
 - Browser version

## Additional context
Tried on a localhost during yesterday's JUG Rotterdam meeting, and today on 2 different hosting environments.
It makes no difference whether htacces.txt is renamed to .htaccess or not.

**Comment by nikosdion** [comment]
I actually need what is _above_ what you have in your screenshot, i.e. the exact error message and the code where it comes from. What you've provided in the screenshot is further down the backtrace which is kinda useless, as all it tells me is that some error happens sometime when trying to show a page — that's pretty much like saying water's wet.

Also, for what it's worth, it sounds like it's something specific to whatever configuration you have on your server. I am looking at a just installed Panopticon 1.0.3 which installed and works perfectly fine. The additional error message context will very likely help me understand what happened with your server.

**Reply by EricdeWaal** [reply]
Hi Nicholas,

Well, that's the point: the screen is very high and shows almost nothing, please see the screenshots 2-5.
![panopticon5](https://github.com/akeeba/panopticon/assets/5586423/89de8517-0363-4f69-a134-d1c1e03ff276)
![panopticon4](https://github.com/akeeba/panopticon/assets/5586423/6044298d-ea1e-4e46-ad88-658e50701441)
![panopticon3](https://github.com/akeeba/panopticon/assets/5586423/33a557d1-b3c6-454b-b428-2e8d8b05411c)
![panopticon2](https://github.com/akeeba/panopticon/assets/5586423/61ac2325-0ebe-409f-9574-bbb87daf6c43)
That's why I also attach the source html
[panopticon-html.zip](https://github.com/akeeba/panopticon/files/13304869/panopticon-html.zip)
l hope that gives you a clue what goes wrong.

I also added the server configuration; nothing special and I get the same error on 2 different hosting providers, and a local installation.
[phpconfig.txt](https://github.com/akeeba/panopticon/files/13305020/phpconfig.txt)

Best,
Eric

Edit: I can send you a link and ftp access in a private message

**Reply by nikosdion** [reply]
Something tells me you are **NOT** using the English language and some translator has cocked it up. Which language are you using?

**Comment by EricdeWaal** [comment]
I am Dutch, and indeed see the first page in Dutch, but see no option to choose an other language

**Reply by nikosdion** [reply, marked as answer]
Change the default language of your browser to English. Does that let you go through the setup?

**Comment by EricdeWaal** [comment]
Yes! That's it. Now I get the second screen. Unbelievable that this language issue can lead to such a huge error!

**Reply by nikosdion** [reply]
That's PHP 8 for you. It elevated small typos to fatal errors.

**Comment by EricdeWaal** [comment]
Well, thanks for solving it, I am now giving Panopticon a try
